internal/wire/parse.go

```diff
@@ -732,6 +739,43 @@ func funcOutput(sig *types.Signature) (outputSignature, error) {
 	}
 }
 
+// processStructLitProvider creates a provider for a named struct type.
+// It produces pointer and non-pointer variants via two values in Out.
+//
+// This is a copy of the old processStructProvider, which is deprecated now.
+func processStructLitProvider(fset *token.FileSet, typeName *types.TypeName) (*Provider, []error) {
+	fmt.Println("Deprecated: use wire.Struct to inject a struct instead,",
```

Comment:
Two things here:

1.  A deprecation notice should go to stderr to stay in the same stream as the other error messages.
2.  This deprecation message does not include enough information for the user to find the element that needs upgrading. Let's include at least the type name, and preferably the file/line number.

Reply:
Done.